index.bs

```diff
@@ -1307,13 +1340,20 @@ these steps:
   1. Optionally, wait until any prior execution of this algorithm has terminated.
 
   1. Display a prompt to the user requesting that the user pick exactly one file.
-
      The displayed prompt should let the user pick one of the |accepts options| to filter the list of displayed files.
      Exactly how this is implemented, and what this prompt looks like is left up to the user agent.
      If |accepts options| are displayed in the UI, the selected option should also be used to suggest an extension
      to append to a user provided file name, but this is not required. In particular user agents are free to ignore
      potentially dangerous suffixes such as those ending in `".lnk"` or `".local"`.
 
+     When possible, this prompt should start out showing |starting directory|.
+
+     If |options|.{{SaveFilePickerOptions/suggestedName}} is specified, the file picker prompt will be pre-filled with the
+     |options|.{{SaveFilePickerOptions/suggestedName}} as the default name to save as. The interaction between the
+     {{SaveFilePickerOptions/suggestedName}} and |accepts options| is left up to the user agent.
```

Comment:
"is left to the user agent" might warrant a non-normative note giving an example of what a UA might do. Not critical though.